webapp/src/AppComponents/DeleteWorkbookDialog.tsx

```diff
@@ -1,21 +1,24 @@
 import styled from "@emotion/styled";
 import { Trash2 } from "lucide-react";
-import { forwardRef, useEffect } from "react";
-import { theme } from "../theme";
-
-export const DeleteWorkbookDialog = forwardRef<
-  HTMLDivElement,
-  {
-    onClose: () => void;
-    onConfirm: () => void;
-    workbookName: string;
-  }
->((properties, ref) => {
```

Comment:
We are using functions in all these things, is there any particular reason you went on a different route?
I might be missing something though.